Dangerfile

```diff
@@ -36,7 +36,7 @@ if github.branch_for_base == 'release'
   # Format message
   message = "### This deploy contains the following PRs:\n\n"
   pr_info.each do |pr_id, info|
-    message << "#{info[:title]} (#{info[:href]})\n"
+    message << "- #{info[:title]} (#{info[:href]})\n"
```

Comment:
This winds up formatting a little nicer with bullet points:

<img width="717" alt="screen shot 2017-02-15 at 12 34 08 pm" src="https://cloud.githubusercontent.com/assets/1457859/22986909/0b1c0540-f37b-11e6-9635-2a4c034fdde0.png">